domain/build.gradle.kts

```
@@ -0,0 +1,9 @@
+plugins {
+    id("java-library")
+    id("org.jetbrains.kotlin.jvm")
```

```suggestion
    id("kotlin")
```
순수 코틀린 모듈로 선언해주려면 위의 플러그인을 선언해주어야합니다!

---

기본생성으로 처리하였는데 조언감사합니다.